__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../app");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+const request = require("supertest");
+
+beforeEach(() => seed(testData));
+afterAll(() => {
+  db.end();
+});
+
+describe("/api/topics", () => {
+  it("GET - 200: Should return an array of 'topic objects'", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.topics)).toBe(true);
+        expect(body.topics[0]).toMatchObject({
```

Comment:
You're checking that the first item in the array conforms to the correct structure, but be careful of making tests that are brittle. Ideally, you would use a forEach loop here, checking against array length, just to make sure that every item in the array is correct and that all items are returned, not just one.

Reply:
Sounds good i'll keep it in mind for the next tests and refactor this, thanks.